styles/mixins/tooltip.js

```diff
@@ -0,0 +1,35 @@
+module.exports = function () {
+  return function (mixin, color) {
+    var color = color || 'rgba(17,17,17,.9)';
```

Comment:
The only issue was there was a lot of complex nested and duplicate selectors and I wasn't sure how to do the mixins like that.  For example:

https://github.com/firstandthird/clientkit/pull/39/files#diff-cb4b362339a1552ea99c0267018a7fd6R126